scenarios/mongoose_muc_light.erl

```diff
@@ -0,0 +1,304 @@
+-module(mongoose_muc_light).
+
+-behavior(amoc_scenario).
+
+-include_lib("exml/include/exml.hrl").
+-include_lib("escalus/include/escalus.hrl").
+-include_lib("escalus/include/escalus_xmlns.hrl").
+
+-required_variable({'ROOM_SIZE', <<"Number of users in a room.">>}).
+-required_variable({'ROOM_CREATION_RATE', <<"Rate of room creations (per minute, def:600)">>}).
+-required_variable({'PUBLICATION_SIZE', <<"Size of additional payload (bytes, def:300)">>}).
+-required_variable({'PUBLICATION_RATE', <<"Rate of publications (per minute, def:1500)">>}).
+-required_variable({'ACTIVATION_POLICY', <<"Publish after subscribtion of (def: all_rooms | n_rooms)"/utf8>>}).
+-required_variable({'MIM_HOST', <<"The virtual host served by the server (def: <<\"localhost\">>)"/utf8>>}).
+-required_variable({'MUC_HOST', <<"The virtual MUC host served by the server (def: <<\"muclight.localhost\">>)"/utf8>>}).
+
+-define(ALL_PARAMETERS, [
+    {room_size, 'ROOM_SIZE', 5, positive_integer},
+    {room_creation_rate, 'ROOM_CREATION_RATE', 600, positive_integer},
+    {publication_size, 'PUBLICATION_SIZE', 300, nonnegative_integer},
+    {publication_rate, 'PUBLICATION_RATE', 1500, positive_integer},
+    {activation_policy, 'ACTIVATION_POLICY', all_rooms, [all_rooms, n_rooms]},
+    {mim_host, 'MIM_HOST', <<"localhost">>, bitstring},
+    {muc_host, 'MUC_HOST', <<"muclight.localhost">>, bitstring}
+]).
+
+-define(GROUP_NAME, <<"coordinator">>).
+-define(ROOM_CREATION_THROTTLING, room_creation).
+-define(PUBLICATION_THROTTLING, publication).
+
+-define(COORDINATOR_TIMEOUT, 100000).
+
+-export([init/0, start/2]).
+
+-spec init() -> {ok, amoc_scenario:state()} | {error, Reason :: term()}.
+init() ->
+    init_metrics(),
+    case config:get_scenario_settings(?ALL_PARAMETERS) of
+        {ok, Settings} ->
+            config:store_scenario_settings(Settings), config:dump_settings(),
+
+            [PublicationRate, RoomCreationRate] = [proplists:get_value(Key, Settings) ||
```

Comment:
Lists comprehension looks a bit like an overkill here maybe just:
```erlang
PublicationRate = proplists:get_value(publication_rate, Settings),
NodeCreationRate = proplists:get_value(node_creation_rate, Settings),
```
??

Reply:
Yeah, I thought the same, but to be honest I copied this part from a different test...

Reply:
And yes, changes are the same as in #81.

Reply:
@asas12 yes, and I asked about the same in the other PR ;P
https://github.com/esl/amoc/pull/81#discussion_r307619242